style.css

```diff
@@ -0,0 +1,86 @@
+.nav {
+  display: flex;
+  justify-content: space-between;
+  width: 330px;
+  align-items: center;
+  text-align: center;
+  height: 20px;
+  margin-bottom: 20px;
+  margin-left: 24px;
+}
+
+.hamburger {
+  float: right;
+  width: 18px;
+  height: 12px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  cursor: pointer;
+}
+
+.bar {
+  width: 16px;
+  height: 2px;
+  background-color: #6070ff;
+  border-radius: 2px;
+}
+
+.logo {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.headline {
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 700;
+  margin-top: 128px;
+  width: 310px;
+  height: 150px;
+  margin-left: 24px;
+}
+
+.content {
+  font-family: Arial, Helvetica, sans-serif;
+  display: flex;
+  flex-direction: column;
+  background-image: url("img/header-shapes mobile@2x.svg");
+  background-repeat: no-repeat;
+  background-size: 100%;
+  font-weight: 200;
+  font-size: 16px;
+  line-height: 24px;
+  width: 375px;
+  height: 100vh;
+}
```

Comment:
There are some few disparities between your design and that of the figma. Kindly check and fix the white/blank space on the right side of your design. Also check the fonts and spacing to make sure it matches the design.

 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-28 05-09-53](https://user-images.githubusercontent.com/65233860/198510340-bc0d6c87-c2e9-4e0c-b48f-5e00feaf82b6.png)  |  ![Screenshot from 2022-10-28 05-25-32](https://user-images.githubusercontent.com/65233860/198510369-78d07027-522b-416d-acff-2d29123d8dc4.png)